pkg/infrastructure/services/infinispan.go

```diff
@@ -157,7 +157,7 @@ func deployInfinispanWithKogitoInfra(instance v1alpha1.InfinispanAware, namespac
 		return false, 0, nil
 	}
 
-	// Overrides any parameters not set
+	// instanceOverrides any parameters not set
```

Comment:
I didn't understand this change :thinking:

Reply:
Ah sorry, I must've copied `instance` and paste here by mistake, I'll change it back